components/ReviewTransactionModal.tsx

```diff
@@ -113,12 +117,22 @@ const ReviewTransactionModal = ({
               </Flex>
             </Flex>
           </Flex>
+          {!shieldPrivateKey && (
+            <Alert status="info" mt=".5rem">
```

Comment:
It would probably look better with a border radius. But that may be personal preference